src/Atmos/Model/turbulence.jl

```diff
@@ -190,7 +190,8 @@ function turbulence_tensors(m::SmagorinskyLilly, state::Vars, diffusive::Vars, a
   # squared buoyancy correction
   Richardson = diffusive.turbulence.N² / (normS^2 + eps(normS))
   f_b² = sqrt(clamp(1 - Richardson*inv_Pr_turb, 0, 1))
-  ν = normS * f_b² * FT(m.C_smag * aux.turbulence.Δ)^2
+  ν = normS * FT(m.C_smag * aux.turbulence.Δ)^2
+  ν = SDiagonal(ν,ν,ν * f_b²)
```

Comment:
This only works in LES configurations where vertical is 3-direction. Please make this vector invariant, using `vertical_unit_vector` to modify the vertical component: Something like `SDiagonal(ν,ν,ν) \otimes f_b² vertical_unit_vector` where \otimes is the Hadamard (component wise) product.